app/js/features/escrow/reducer.js

```diff
@@ -40,6 +40,11 @@ const escrowBuilder = function (escrowObject) {
 
 function reducer(state = DEFAULT_STATE, action) {
   let escrows  = cloneDeep(state.escrows);
+  let currentEscrow;
+  if (action.escrowId) {
+    currentEscrow = escrows.find(escrow => escrow.escrowId === action.escrowId);
+  }
+
```

Comment:
What if currentEscrow is undefined at this stage (line 47)?
Is it even possible?

Reply:
It's possible that escrowId is available but it's a new escrow (eg creating a new escrow), but the reducer case doesn't use `currentEscrow` so it's not an issue there. 
We could add a condition on the cases that require `currentEscrow` and throw an error, but that's what's gonna happen anyway.